packages/core/src/routes/connector.ts

```diff
@@ -131,61 +126,6 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
     }
   );
 
-  router.patch(
-    '/connectors/:id/enabled',
-    koaGuard({
-      params: object({ id: string().min(1) }),
-      body: Connectors.createGuard.pick({ enabled: true }),
-    }),
-    async (ctx, next) => {
-      const {
-        params: { id },
-        body: { enabled },
-      } = ctx.guard;
-
-      const {
-        type,
-        dbEntry: { config },
-        metadata,
-        validateConfig,
-      } = await getLogtoConnectorById(id);
-
-      if (enabled) {
-        validateConfig(config);
-      }
-
-      // Only allow one enabled connector for SMS and Email.
-      // disable other connectors before enable this one.
-      if (enabled && (type === ConnectorType.Sms || type === ConnectorType.Email)) {
-        const connectors = await getLogtoConnectors();
-        await Promise.all(
-          connectors
-            .filter(
-              ({ dbEntry: { enabled }, type: currentType }) => type === currentType && enabled
-            )
-            .map(async ({ dbEntry: { id } }) =>
-              updateConnector({ set: { enabled: false }, where: { id }, jsonbMode: 'merge' })
-            )
-        );
-      }
-
-      const connector = await updateConnector({
-        set: { enabled },
-        where: { id },
-        jsonbMode: 'merge',
-      });
-
-      // Delete the social connector in the sign-in experience if it is disabled.
-      if (!enabled && type === ConnectorType.Social) {
```

Comment:
need to implement this on "delete" API before merging this PR?

Reply:
Move this logic to [this PR](https://github.com/logto-io/logto/pull/2524).